src/java.base/share/classes/java/util/zip/DeflaterInputStream.java

```diff
@@ -34,6 +34,19 @@
  * Implements an input stream filter for compressing data in the "deflate"
  * compression format.
  *
+ * <h2 id="compressor-usage">Compressor Usage</h2>
+ * An {@linkplain DeflaterInputStream input stream} that is created without specifying
+ * a {@linkplain Deflater compressor} will use a default compressor. The default
```

Comment:
Reading this again makes it wonder if "will use a default compressor" should be adjusted to say "will create a compressor", only to avoid anyone thinking there is a system wide default compressor.